pyteal/ast/subroutine.py

```diff
@@ -269,15 +292,39 @@ def has_return(self):
 SubroutineDeclaration.__module__ = "pyteal"
 
 
+@dataclass
+class _OutputKwArgInfo:
+    name: str
+    abi_type: abi.TypeSpec
+
+    @staticmethod
+    def from_dict(kwarg_info: dict[str, abi.TypeSpec]) -> Optional["_OutputKwArgInfo"]:
+        if kwarg_info is None or len(kwarg_info) == 0:
+            return None
+        elif len(kwarg_info) == 1:
+            key = list(kwarg_info.keys())[0]
```

Comment:
```suggestion
            (key) = [*kwarg_info]
```
lil destructuring hack... feel free to ignore

Reply:
resolved by #308